static/directives/org-binding.html

```diff
@@ -1,32 +1,32 @@
 <div class="org-binding-settings-element" >
-  <span><h3>Monthly Subscriptions From Red Hat Customer Portal</h3></span>
+  <span><h3>Subscriptions From Red Hat Customer Portal</h3></span>
   <div class="cor-loader-inline" ng-show="marketplaceLoading"></div>
   <span ng-show="!organization && !marketplaceLoading">
     <div ng-repeat="subscription in userMarketplaceSubscriptions">
-      {{subscription.quantity}}x {{ subscription.sku }}
+      {{subscription.quantity}}x {{ subscription.metadata.privateRepos }} private repos
```

Comment:
do we always get `metadata` and `privateRepos`?

Reply:
`metadata` should be always be there for user and org skus. As for `privateRepos` it should always be part of metadata since `metadata` is the equivalent [plan](https://github.com/quay/quay/blob/master/data/billing.py#L12) that we have hardcoded in the billing section